sippy-ng/src/component_readiness/CompReadyVars.js

```diff
@@ -240,6 +264,127 @@ export const CompReadyVarsProvider = ({ children }) => {
     setCapability(capabilityParam)
   }
 
+  const initialViews = {
+    Default: {
+      config: {
+        help: 'The most commonly used view and focuses on amd64',
+        'Group By': defaultGroupByCheckedItems,
+        'Exclude Arches': defaultExcludeArchesCheckedItems,
+        'Exclude Networks': defaultExcludeNetworksCheckedItems,
+        'Exclude Clouds': defaultExcludeCloudsCheckedItems,
+        'Exclude Upgrades': defaultExcludeUpgradesCheckedItems,
+        'Exclude Variants': defaultExcludeVariantsCheckedItems,
+        Confidence: defaultConfidenceParam,
+        Pity: defaultPityParam,
+        'Min Fail': defaultMinFailParam,
+        'Ignore Missing': defaultIgnoreMissingParam,
+        'Ignore Disruption': defaultIgnoreDisruptionParam,
+      },
+    },
+    UpgradeVariants: {
+      config: {
+        help: 'Default view but groups by upgrades and variants',
+        'Group By': ['upgrade', 'variants'],
+        'Exclude Arches': defaultExcludeArchesCheckedItems,
+        'Exclude Networks': defaultExcludeNetworksCheckedItems,
+        'Exclude Clouds': defaultExcludeCloudsCheckedItems,
+        'Exclude Upgrades': defaultExcludeUpgradesCheckedItems,
+        'Exclude Variants': defaultExcludeVariantsCheckedItems,
+        Confidence: defaultConfidenceParam,
+        Pity: defaultPityParam,
+        'Min Fail': defaultMinFailParam,
+        'Ignore Missing': defaultIgnoreMissingParam,
+        'Ignore Disruption': defaultIgnoreDisruptionParam,
+      },
+    },
+    ARM64: {
+      config: {
+        help: 'Default view but only for amd64',
+        'Group By': defaultGroupByCheckedItems,
+        'Exclude Arches': ['amd64', 'ppc64le', 's390x', 'heterogeneous'],
```

Comment:
Hm, it'll probably be better once we've got server-side views, since we'll be able to do some kind of "include" version of the filter.  If we add a new arch now then it'll show up in this view along with ARM since filters are excluded.